zip-0225.rst

```diff
@@ -265,8 +264,8 @@ TxId Digest
 
 The tree of hashes defined by ZIP 244 [#zip-0244]_ is re-structured to include a new
 branch for Orchard hashes. The ``orchard_digest`` branch is the only new addition to the
-tree; ``header_digest``, ``transparent_digest``, ``sprout_digest``, and ``sapling_digest``
-are as in ZIP 244::
+tree; ``header_digest``, ``transparent_digest``, and ``sapling_digest`` are as in ZIP
```

Comment:
We're hashing
```
header_digest
transparent_sig_digest
sprout_digest
sapling_digest
```
in v4, and
```
header_digest
transparent_sig_digest
sapling_digest
orchard_digest
```
in v5. Although this doesn't strictly cause any problem (partly because of the personalization on the hashes for each branch, and partly because the tx version number is hashed in the header), it still seems potentially confusing and error-prone. Should we include an empty ``sprout_digest`` in the same place in v5?

Reply:
Separately: the hash of the transparent branch is called ``transparent_sig_digest`` [in ZIP 244](https://zips.z.cash/zip-0244#signature-digest), not ``transparent_digest``.

Reply:
Does it still make sense to change the txid digest algorithm for V4 transactions? Are there meaningful use cases that require a stable txid digest where one can't simply use a V5 transaction? I had thought not, and so I removed the sprout digests. But perhaps that requires additional rework, if this is to apply to V5 transactions only?

Reply:
Oh right, I'd forgotten that the spec that included `sprout_digest` will never actually be deployed. OK, that part looks fine then.